lms/templates/courseware_search/search_list.underscore

```diff
@@ -10,11 +10,11 @@
     <% if (hasMoreResults) { %>
         <a class="search-load-next" href="javascript:void(0);">
             <%= interpolate(
-                ngettext("Load next %(num_items) result", "Load next %(num_items) results", pageSize),
+                ngettext("Load next %(num_items)s result", "Load next %(num_items)s results", pageSize),
```

Comment:
It tells how to format the number inside the %() - the i8ln team changed it to allow for paging by 1, and did not replace the s character, which is the format specifier for the number within the string (I guess 's' is 'formatted as a string')